lib/ansible/modules/storage/ibm/ibm_sa_vol_map.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2018 IBM CORPORATION
+# Author(s): Tzur Eliyahu <tzure@il.ibm.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: ibm_sa_vol_map
+short_description: Handles volume mapping on an IBM Spectrum Accelerate storage array.
+version_added: "2.8"
+
+description:
+    - "This module maps volumes to or unmaps them from the hosts on
+        IBM Spectrum Accelerate storage systems."
+
+options:
+    vol:
+        description:
+            - Volume name.
+        required: true
+    state:
+        default: "present"
+        choices: [ "present", "absent" ]
+        description:
+            - When the state is present the volume is mapped.
+                When the state is absent, the volume is meant to be unmapped.
+        required: true
+    cluster:
+        description:
+            - Maps the volume to a cluster.
+        required: false
+    host:
+        description:
+            - Maps the volume to a host.
+        required: false
+    lun:
+        description:
+            - The LUN identifier.
+        required: false
+    override:
+        description:
+            - Overrides the existing volume mapping.
+        required: false
+
+extends_documentation_fragment:
+    - ibm_storage
+
+author:
+    - Tzur Eliyahu (tzure@il.ibm.com)
+'''
+
+EXAMPLES = '''
+- name: Map volume to host.
+  ibm_sa_vol_map:
+    vol: volume_name
+    lun: 1
+    host: host_name
+    username: admin
+    password: secret
+    endpoints: hostdev-system
+    state: present
+
+- name: Map volume to cluster.
+  ibm_sa_vol_map:
+    vol: volume_name
+    lun: 1
+    cluster: cluster_name
+    username: admin
+    password: secret
+    endpoints: hostdev-system
+    state: present
+
+- name: Unmap volume.
+  ibm_sa_vol_map:
+    host: host_name
+    username: admin
+    password: secret
+    endpoints: hostdev-system
+    state: absent
+'''
+RETURN = '''
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ibm_sa_utils import execute_pyxcli_command, \
```

Comment:
Rather than using a backslash continuation for multiple imports, use parentheses:

```suggestion
from ansible.module_utils.ibm_sa_utils import (
    execute_pyxcli_command,
    connect_ssl,
    spectrum_accelerate_spec,
    is_pyxcli_installed,
)
```

See [PEP 328](https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses) for rationale.

Reply:
will fix

Reply:
Regarding the new modules - We've tested this internally with our arrays. Same as we tested the other modules we contributed in Ansible 2.7